deploy/helm/sumologic/README.md

```diff
@@ -482,6 +478,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `pvcCleaner.job.podAnnotations`                                                       | Additional annotations for for the pvcCleaner container.                                                                                                                                                                                                                                                   | `{}`                                                                                                                                                                                 |
 | `pvcCleaner.job.schedule`                                                             | Schedule for cronJobs                                                                                                                                                                                                                                                                                      | `*/15 * * * *`                                                                                                                                                                       |
 | `pvcCleaner.job.securityContext`                                                      | The securityContext configuration for the pvcCleaner.                                                                                                                                                                                                                                                      | `{"runAsUser": 1000}`                                                                                                                                                                |
+| `debug.instrumentation.tracesGateway.print`                                           | Set to `true` to print logs processed by log collector on standard output.                                                                                                                                                                                                                                 | `false`                                                                                                                                                                              |
```

Comment:
Please add `stopLogsIngestion` option as well. It's important to prevent any kind of logs flooding